Get-help.md

```diff
@@ -48,13 +43,44 @@ If you have a question about a pull request or issue, you can also reach out to
+* [oppia-dev](https://groups.google.com/forum/#!forum/oppia-dev) is the main mailing list for communication between developers and for technical questions. You can post to it even if you're not a member of the group. This is where you can ask questions, solicit feedback, or make developer-specific announcements (e.g. a temporary GitHub outage).
+
+You can also email your mentor with any questions. If you don't have a mentor, complete the steps on the [[wiki page for contributing code to Oppia|Contributing-code-to-Oppia]] and we'll try to assign you.
+
 ## How to Ask Good Questions
 
-At Oppia we don’t care how silly your question is! Just ensure your question is clear, and provide us with enough information to help us resolve it faster. We've divided the questions into 2 categories - Setup-related and General questions. You can start following the sections below to understand how you can ask each of them.
+At Oppia we don’t care how silly your question is! Just ensure your question is clear, and provide us with enough information to help us resolve it faster. 
+
+### General guidelines to follow 
+
+* __Be clear and concise__: Clearly articulate your question to avoid confusion and allow others to understand it easily.
+* __Provide context__: If applicable, provide relevant background information to help others understand your question and situation.
+* __Be respectful__: Maintain a respectful and courteous tone in your communication to foster a positive and inclusive community environment.
+* __Format your text for easy reading__: Organize your question using paragraphs, bullet points, and appropriate formatting to enhance readability.
+* __Proofread your message__: Take a moment to proofread your question to ensure clarity and accuracy before sending it.
+
+### Providing error information 
+
+When reporting an error, follow these steps to communicate the issue effectively:
+
+* __Start with the command__: Rather than saying "I have an error," provide the actual command you ran. Copy and paste the command in your email or chat message to give recipients the complete context.
+* __Include the specific error message__: Share the exact error message you encountered. This helps others quickly identify the potential cause and provide relevant solutions or suggestions.
+* __Share reproduction steps__: Include the specific steps that recipients should follow to reproduce the error you encountered. This allows them to investigate the issue more effectively.
+* __Explain your efforts and conclusions__: Explain what you've tried and what conclusions you've drawn, if any. (Providing a [[Debugging Docs|Debugging-Docs]] may help.)
+* __Including relevant log lines__: Sometimes, log lines from the browser console or terminal can provide helpful context. However, ensure to only include relevant lines and omit unnecessary information to maintain the readability of the report.
```

Comment:
This looks great! I just have a couple suggestions

```suggestion
* __Include relevant log lines__: Sometimes, log lines from the browser console or terminal can provide helpful context. However, ensure you only include relevant lines and omit unnecessary information to maintain the readability of the report.
```

Reply:
done